packages/react-theme/tsconfig.json

```diff
@@ -1,18 +1,25 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["src"],
   "compilerOptions": {
     "target": "ES2019",
-    "module": "CommonJS",
-    "lib": ["ES2019", "DOM"],
-    "outDir": "dist",
-    "jsx": "react",
-    "declaration": true,
-    "experimentalDecorators": true,
+    "noEmit": true,
+    "isolatedModules": true,
     "importHelpers": true,
+    "jsx": "react",
     "noUnusedLocals": true,
-    "preserveConstEnums": true,
-    "types": ["jest", "custom-global", "inline-style-expand-shorthand"],
-    "isolatedModules": true
-  }
+    "preserveConstEnums": true
+  },
+  "include": [],
+  "files": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },
+    {
+      "path": "./tsconfig.spec.json"
+    },
+    {
+      "path": "./.storybook/tsconfig.json"
+    }
```

Comment:
```suggestion
    }
```

Reply:
oh wait, you didn't remove storybook... please update the PR description

Reply:
This package contains stories, so we should keep this config